requirements_dev.txt

```diff
@@ -4,3 +4,8 @@ gemmi==0.4.6
 graphviz==0.16
 pytest==6.2.4
 pytest-cov==2.12.0
+mrcfile==1.3
+workflows==2.9
+pillow==8.0
+numpy==1.21; python_version >= '3.7'
+numpy==1.19; python_version < '3.7'
```

Comment:
These (except `workflows`) will need to be replicated in `requirements.txt`, possibly in `requirements_doc.txt` (we'll see on merging). Packages also need adding to `setup.cfg`'s `install_requires`, but without the version constraint.